fetch.bs

```diff
@@ -5961,25 +5961,68 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
    <li><p>Return the <a for=/>appropriate network error</a> for <var>fetchParams</var>.
   </ol>
 
- <li><p>Let <var>pullAlgorithm</var> be an algorithm that <a lt=resumed for=fetch>resumes</a> the
- ongoing fetch if it is <a lt=suspend for=fetch>suspended</a>.
+ <li>
+  <p>Let |buffer| be an empty buffer that can have bytes appended to it.
+
+  <p class="note">This represents an internal buffer inside the network layer of the user agent.
+
+ <li><p>Let |pullAlgorithm| be the followings steps:
+
+  <ol>
+   <li><p>Let |promise| be [=a new promise=].
+
+   <li><p>Run the following steps [=in parallel=]:
+
+    <ol>
+     <li><p>If the size of |buffer| is smaller than a lower limit chosen by the user agent and the
+     ongoing fetch is [=fetch/suspend|suspended=], [=fetch/resumed|resume=] the fetch.
+
+     <li><p>Wait until |buffer| is not empty.
+
+     <li><p>[=Queue a fetch task=] to run the following steps, with |fetchParams|'s [=task destination=].
+
+      <ol>
+       <li><p>Let |available| be the size of |buffer|.
+
+       <li><p>Let |desiredSize| be |available|.
+
+       <li><p>If |stream|'s [=ReadableStream/current BYOB request view=] is non-null, then set
+       |desiredSize| to |stream|'s [=ReadableStream/current BYOB request view=]'s [=BufferSource/byte
+       length=].
```

Comment:
No newlines inside terms.

Reply:
Hmm, that's not what the Great Rewrapper does, but okay.

Reply:
If that's Dominic's tool that's a known issue I think. HTML has a different convention that cares less about inline search. I want inline search to be as straightforward as possible.